### PR TITLE
Done some changes for quickstart and sql docs

### PR DESCRIPTION
**Pull request by ad1happy2go**
Quickstart Updates - 
1. CDC query for data source
2. Delete python example is broken - Fixed.
3. Write a payload example for Merging Data section. MIT example can show a more complex merge expression? - Done
4. Does MIT, Delete, update Scala/python quick start example need precombine? Do we need precombine file in datasource write operation examples?
      - Quickstart Doesn't need as it has default "ts" column.
5. Consistent table names and use of data across DataFrame vs SQL

### CODE REVIEW

**Review by bhasudha — APPROVED** [review]
lgtm. We discussed couple changes to be sent in another patch. from @ad1happy2go

File: website/docs/sql_ddl.md [hunk 1]
```diff
@@ -204,6 +204,28 @@ Users can set table properties while creating a table. The important table prope
...
+```
+
+### Enabling Data Skipping Using RLI
+Users can enable Record Level index using the following table properties.
+
+```sql
+TBLPROPERTIES(
+            'hoodie.metadata.enable' = 'true',
+            'hoodie.metadata.record.index.enable' = 'true',
+            'hoodie.enable.data.skipping' = 'true'
```
Comment (vinothchandar) [inline website/docs/sql_ddl.md]: is n't this a read side config?

File: website/docs/sql_ddl.md [hunk 2]
```diff
@@ -204,6 +204,28 @@ Users can set table properties while creating a table. The important table prope
 `primaryKey`, `preCombineField`, and `type` and other properties are case-sensitive. 
 :::
 
+### Enabling Data Skipping Using Column Stats
+Users can enable column stats index using the following table properties.
+
+```sql
+TBLPROPERTIES(
+            'hoodie.metadata.enable' = 'true',
```
Comment (vinothchandar) [inline website/docs/sql_ddl.md]: this is true by default? so avoid calling it out explicitly?

**Review by vinothchandar — COMMENTED** [review]
One nit.

File: website/docs/sql_dml.md [hunk 3]
```diff
@@ -184,7 +184,7 @@ DELETE FROM hudi_cow_pt_tbl WHERE name = 'a1';
 These DML operations give you powerful tools for managing your Hudi tables using Spark SQL. 
 You can control the behavior of these operations using various configuration options, as explained in the documentation.
 
-### Passing Options 
+### Passing Options [Should we remove ???]
```
Comment (vinothchandar) [inline website/docs/sql_dml.md:187]: yeah probably.. we can show using `SET ` ?

**Review by vinothchandar — COMMENTED** [review]
LGTM . I am incorporating the changes and push to the PR

**Pull request closed without merging**